modules/ztp-acm-installing-disconnected-rhacm.adoc

```diff
@@ -15,6 +15,11 @@ Use {rh-rhacm-first}, {gitops-title}, and {cgu-operator-first} on the hub cluste
 * You have logged in as a user with `cluster-admin` privileges.
 
 * You have configured a disconnected mirror registry for use in the cluster.
++
+[NOTE]
+====
+The disconnected mirror registry that you create must contain an image version that matches the version of {cgu-operator-first} running in the hub cluster. The disconnected mirror registry that you specify must be accessible from the spoke cluster.
```

Comment:
I think this needs a bit more context. The images specified here are the precaching and backup job images. Something like this perhaps?
```suggestion
The disconnected mirror registry that you create must contain a version of TALM backup and precache images that match the version of {cgu-operator-first} running in the hub cluster. The spoke cluster must be able to resolve these images in the disconnected mirror registry.
```

Reply:
ok. have changed accordingly